tests/util/helpers.sh

```diff
@@ -55,7 +55,7 @@ _wait_for_istio() {
     local kind="$1"
     local namespace="$2"
     local name="$3"
-    if ! istioctl experimental wait --for=distribution --timeout=30s "$kind" "$name.$namespace"; then
+    if ! istioctl experimental wait --for=distribution --timeout=1m "$kind" "$name.$namespace"; then
```

Comment:
Why not just make it 5m, same as _wait_for_deployment(). It's just a max wait, so if it should never happen.
```suggestion
    if ! istioctl experimental wait --for=distribution --timeout=5m "$kind" "$name.$namespace"; then
```

Reply:
@frankbu Because there is a problem where occasionally the test waits the full timeout even if the requirements are satisfied. So we had multiple 5 min waits happening across the tests, and then we would blow the 60 min timeout as well. Maybe that's not an issue anymore. Need to look at the tests again.

Reply:
I looked at the test results for the three tests here, and don't see any timeout failures. Maybe we can go back to the 5m as you suggest and turn the failure back on if we timeout and see what happens.  Commit upcoming.